java/struts2/pom.xml

```diff
@@ -5,11 +5,12 @@
     <groupId>org.apache.struts.benchmarker</groupId>
     <artifactId>struts2</artifactId>
     <version>1.0</version>
-    <packaging>war</packaging>
+    <packaging>jar</packaging>
```

Comment:
Just for information, what is the difference between `war` and `jar` deployment method ?

Also, is the any universal deployment type, like for `sbt` in **scala** ?
https://github.com/the-benchmarker/web-frameworks/blob/master/scala/Dockerfile#L7

Reply:
`war` is used when you want deploy the package into a Java servlet container (like jetty-runner or standalone Jetty server). `jar` is a common Java package used for libraries and apps (executable). If you want to start your app with `java -jar` it has to be a proper jar. I replaced `war` with `jar` as `war` is not needed any more.

No, you must use maven-assembly-plugin, btw. `universal` is also a plugin https://www.scala-sbt.org/sbt-native-packager/formats/universal.html